Makefile

```diff
@@ -70,7 +70,9 @@ docker:
 	docker build --tag="shopify/toxiproxy:git" .
 
 docker-release:
-	docker build --tag="shopify/toxiproxy:$(VERSION)" .
+	mkdir -p tmp
+	docker run -it --rm -v `pwd`/tmp:/go/bin -v `pwd`:/go/src/github.com/Shopify/toxiproxy golang:1.7 sh -c 'go build -ldflags="-X github.com/Shopify/toxiproxy.Version=$(cat /go/src/github.com/Shopify/toxiproxy/VERSION)" -o /go/bin/toxiproxy github.com/Shopify/toxiproxy/cmd && go build -o /go/bin/toxiproxy-cli github.com/Shopify/toxiproxy/cli'
```

Comment:
Would it be possible to just copy `tmp/build/toxiproxy-server-linux-amd64` instead? Or is there something weird about the alpine linux container that's different?